src/dex/index.ts

```diff
@@ -41,13 +40,12 @@ import { KyberDmm } from './kyberdmm/kyberdmm';
 import { Platypus } from './platypus/platypus';
 import { GMX } from './gmx/gmx';
 import { WooFi } from './woo-fi/woo-fi';
-import { Dystopia } from './uniswap-v2/dystopia/dystopia';
-import { SpiritSwapV2 } from './spirit-swap-v2';
 import { ParaSwapLimitOrders } from './paraswap-limit-orders/paraswap-limit-orders';
 import { AugustusRFQOrder } from './augustus-rfq';
 import Web3 from 'web3';
-import { Velodrome } from './velodrome';
-import { Solidly } from './solidly';
+import { Solidly } from './solidly/solidly';
+import { Velodrome } from './solidly/forks-override/velodrome';
+import { SpiritSwapV2 } from './solidly/forks-override/spiritSwapV2';
```

Comment:
I think you just need to import only main implementation, for example, Solidy (or Dystopia). Other forks should be picked up and added from config automatically from function:

```
  public static dexKeysWithNetwork: { key: string; networks: Network[] }[] =
    getDexKeysWithNetwork(SynthetixConfig);

```

Reply:
Nop it's purposeful, mind:
- omit here  https://github.com/paraswap/paraswap-dex-lib/blob/e971fb70bfe8dd4e94f23ad7c5975df6921a9a0e/src/dex/solidly/solidly.ts#L34
- pick here https://github.com/paraswap/paraswap-dex-lib/blob/e971fb70bfe8dd4e94f23ad7c5975df6921a9a0e/src/dex/solidly/forks-override/spiritSwapV2.ts#L24

Though echoes your other comment about how to implement propre override.